Content.Server/StationGoal/StationGoalCommand.cs

```diff
@@ -0,0 +1,35 @@
+using Content.Server.Administration;
+using Content.Server.StationGoal.Systems;
+using Content.Shared.Administration;
+using Robust.Shared.Console;
+using JetBrains.Annotations;
+
+namespace Content.Server.StationGoal
+{
+    /// <summary>
+    ///     Admin command to set station goal by id.
+    /// </summary>
+    [UsedImplicitly]
+    [AdminCommand(AdminFlags.Fun)]
+    public sealed class StationGoalCommand : IConsoleCommand
+    {
+        public string Command => "setstationgoal";
+        public string Description => "Send station goal papers codes to the communication console";
+        public string Help => "setstationgoal";
```

Comment:
Нужно отразить в подсказе команды о том, что она принимает обязательный параметр.
```suggestion
        public string Help => "setstationgoal <id>";
```